pkg/router/template/router.go

```diff
@@ -512,12 +512,27 @@ func (r *templateRouter) writeConfig() error {
 		r.state[k] = cfg
 	}
 
+	data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+	if err != nil {
+		glog.Warningf("Error reading ARP neighbour information: %s", err)
+	}
+	arpcache, err := strconv.Atoi(string(data[:len(data)-1]))
+	if err != nil {
+		glog.Warningf("Error: %s", err)
+	}
+	arpthreshold := float64(arpcache) * 0.9
```

Comment:
Same here, we should compute arpthreshold only when arpcache is valid (i.e. when strconv.Atoi() returns no error)